### ISSUE TITLE
Broken example

### ISSUE THREAD

**Issue by dtwardy** [vercel/next.js#22035]
**What example does this report relate to?**

with-graphql-react

**What version of Next.js are you using?**

10.0.6

**What version of Node.js are you using?**

14.15.4

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows 10 20H2

**How are you deploying your application?**

npm run dev

**Describe the Bug**

Simply shows "Error!".

Manual build process through ```npm run build``` fails as well.

**Expected Behavior**

Not getting an error when simply starting the example

**To Reproduce**

Simply run ```npx create-next-app --exmaple with-graphql-react```

The script started with ```npm run build``` fails with the following log:

```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/home/XXX/.nvm/versions/node/v14.15.4/bin/node',
1 verbose cli   '/home/XXX/.nvm/versions/node/v14.15.4/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build'
1 verbose cli ]
2 info using npm@6.14.10
3 info using node@v14.15.4
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle with-graphql-react@~prebuild: with-graphql-react@
6 info lifecycle with-graphql-react@~build: with-graphql-react@
7 verbose lifecycle with-graphql-react@~build: unsafe-perm in lifecycle true
8 verbose lifecycle with-graphql-react@~build: PATH: /home/XXX/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/var/www/nxtjs_graph_react/node_modules/.bin:/home/XXX/.rvm/gems/ruby-2.7.2$9 verbose lifecycle with-graphql-react@~build: CWD: /var/www/nxtjs_graph_react
10 silly lifecycle with-graphql-react@~build: Args: [ '-c', 'next build' ]
11 silly lifecycle with-graphql-react@~build: Returned: code: 1  signal: null
12 info lifecycle with-graphql-react@~build: Failed to exec build script
13 verbose stack Error: with-graphql-react@ build: `next build`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/home/XXX/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (/home/XXX/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1048:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid with-graphql-react@
15 verbose cwd /var/www/nxtjs_graph_react
16 verbose Linux 5.4.72-microsoft-standard-WSL2
17 verbose argv "/home/XXX/.nvm/versions/node/v14.15.4/bin/node" "/home/XXX/.nvm/versions/node/v14.15.4/bin/npm" "run" "build"
18 verbose node v14.15.4
19 verbose npm  v6.14.10
20 error code ELIFECYCLE
21 error errno 1
22 error with-graphql-react@ build: `next build`
22 error Exit status 1
23 error Failed at the with-graphql-react@ build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by jamesmosier** [comment]
Hi @dtwardy. I wasn't able to reproduce on Mac OS 11.2 with Node v14 or Node v12. Do any of the other examples work for you? Is there any other info you could provide to help track this down?

**Comment by dtwardy** [comment]
Hi @jamesmosier. Other examples work without a hitch. I noticed, that I did not mention, that the platform is WSL2 (Debian 10 buster) on Win10. Sorry, my bad. The build fails every single time. However it does work on a remote server apparently which is also running Debian 10 buster. Other Next.js examples like ```with-apollo``` or ```with-typescript``` or ```with-mocha``` work in dev mode with ```npm run dev``` but also fail with ```npm run build```.

The build error occurs at the module _sharp_ which apparently cannot be found. I tried ```npm ci``` before starting the build process.

I tried the same approach on 2 different machines with the same result. However, a fresh WSL2 machine running Ubuntu 20.04 LTS did not reproduce the error during build which gets even weired now. My main machines do not run into any issues running the backend and several nextjs apps in dev mode. 

I tried switching between directly installing node and through nvm - on Ubuntu with no difference.

**However** even though the build error did not occur on the fresh Ubuntu machine, the example ended up with an error as well when running it
